lib/diagnostic.rb

```diff
@@ -29,31 +29,31 @@
 
 character = 'Jar Jar Binks'
 
-Diagnostic.interpolation = 'ANSWER is really a Sith Lord.'
+Diagnostic.interpolation = print '#{character} is really a Sith Lord.'
 
 # Numbers in Ruby
 #
 # What is the type of object in Ruby for decimal numbers? Write your answer in
 # a Ruby string.
 
-Diagnostic.decimal = 'your response here'
+Diagnostic.decimal = to_f
```

Comment:
`to_f` will attempt to convert the object that it's invoked upon into a `Float`.  The correct answer is `Float`.